src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -134,6 +151,27 @@ void GlobalAppSettings::SetShowTabsInTitlebar(const bool showTabsInTitlebar) noe
 {
     _showTabsInTitlebar = showTabsInTitlebar;
 }
+
+std::optional<int32_t> GlobalAppSettings::GetInitialX() const noexcept
+{
+    return _initialX;
+}
+
+void GlobalAppSettings::SetInitialX(const int32_t initialX) noexcept
+{
+    _initialX.value() = initialX;

```

Comment:
Why did you change this from "_initialX = initialX"?  Does the new code really work?  It looks like std::optional will throw an exception if .value() is called when it doesn't yet have a value.  Should these use emplace(), like your Parse function does?  It looks like there aren't any callers of SetInitialX/Y -- should these functions be removed if they aren't currently needed/

Reply:
Sorry for this mistake. I should use emplace here. And I removed them because they are not used anywhere.